src/sagemaker/tensorflow/README.rst

```diff
@@ -15,485 +34,130 @@ is a two-step process.
 
 .. code:: python
 
-  def estimator_fn(run_config, hyperparameters):
-      # Defines the features columns that will be the input of the estimator
-      feature_columns = [tf.feature_column.numeric_column(INPUT_TENSOR_NAME, shape=[4])]
-      # Returns the instance of estimator.
-      return tf.estimator.DNNClassifier(feature_columns=feature_columns,
-                                        hidden_units=[10, 20, 10],
-                                        n_classes=3,
-                                        config=run_config)
-
-More details on how to create a ``tensorflow.estimator`` can be find in `Creating Estimators in tf.estimator <https://www.tensorflow.org/extend/estimators>`_.
-
-An example on how to create a TensorFlow training script with an ``estimator_fn`` can find in this `example <https://github.com/awslabs/amazon-sagemaker-examples/tree/master/sagemaker-python-sdk/tensorflow_iris_dnn_classifier_using_estimators>`_.
+    estimator.fit({'train':'s3://my-bucket/my-training-data',
+                  'eval':'s3://my-bucket/my-evaluation-data'})
 
-
-Using a ``Keras`` model instead of a ``model_fn``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-``tf.keras`` is an full implementation inside TensorFlow of the Keras API. To use a ``tf.keras``
-model for training instead of ``model_fn``, you need to write a ``keras_model_fn``. The base skeleton of
-a ``keras_model_fn`` looks like this:
+   In your training script the channels will be stored in environment variables ``SM_CHANNEL_TRAIN`` and
+   ``SM_CHANNEL_EVAL``. You can add them to your argument parsing logic like this:
 
 .. code:: python
 
-  def keras_model_fn(hyperparameters):
-      # Logic to do the following:
-      # 1. Instantiate the Keras model
-      # 2. Compile the Keras model
-      return compiled_model
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--train', type=str, default=os.environ.get('SM_CHANNEL_TRAIN'))
+    parser.add_argument('--eval', type=str, default=os.environ.get('SM_CHANNEL_EVAL'))
 
+3. Export your final model to path stored in environment variable ``SM_MODEL_DIR`` which should always be
+   ``/opt/ml/model``. At end of training SageMaker will upload the model file under ``/opt/ml/model`` to
+   ``output_path``.
 
-Example of a complete ``keras_model_fn``
-''''''''''''''''''''''''''''''''''''''''
 
-.. code:: python
+Calling fit
+^^^^^^^^^^^
 
-  def keras_model_fn(hyperparameters):
-    # Instantiate a Keras inception v3 model.
-    keras_inception_v3 = tf.keras.applications.inception_v3.InceptionV3(weights=None)
-    # Compile model with the optimizer, loss, and metrics you'd like to train with.
-    keras_inception_v3.compile(optimizer=tf.keras.optimizers.SGD(lr=0.0001, momentum=0.9),
-                          loss='categorical_crossentropy', metric='accuracy')
-    return keras_inception_v3
+If you want to use Script Mode, set at least one of these args
```

Comment:
we're assuming at this point in the README that the user does want to use Script Mode, so I think you can remove "If you want"